src/downloads/integration.spec.ts

```
@@ -91,22 +104,21 @@ describe('downloads integration tests', () => {
 
   describe('end-to-end download flow', () => {
     it('should handle complete download lifecycle with electron-dl integration', async () => {
-      // Setup electron-dl mock
+      // Setup downloads system - this will call electronDl
+      setupElectronDlWithTracking();
+      setupDownloads();
+      
```

ESLint

[error] Delete `······`

(prettier/prettier)